src/decorator-annotator.ts

```diff
@@ -148,7 +148,8 @@ export class DecoratorClassVisitor {
    * This method looks for the place where the value that is associated to
```

Comment:
Oh, the user did. But the old code did not. I.e. the old code just looked for the first import that had the right path, which might have been different than the one the user specified.